client/src/components/Notifications/NotificationDropDown.js

```diff
@@ -184,6 +198,17 @@ const MenuItem = ({
   t,
   gtmId,
 }) => {
+  const { clearNotification } = useContext(WebSocketContext); 
+  const dispatch = useDispatch();
+
+  const clearLocalNotification = (notification_id) => {
```

Comment:
Since camelCase is the standard convention for JavaScript, I would change this, and associated references, to `notificationId`